nmt-python-advanced-finetune multilingual-nmt-model-with-nemo.ipynb

```diff
@@ -0,0 +1,871 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2c3dfb21",
+   "metadata": {},
+   "source": [
+    "## NVIDIA NeMo Overview\n",
+    "\n",
+    "NVIDIA NeMo is a toolkit for building new state-of-the-art conversational AI models. NeMo has separate collections for Automatic Speech Recognition (ASR), Natural Language Processing (NLP), and Text-to-Speech (TTS) models. Each collection consists of prebuilt modules that include everything needed to train on your data. Every module can easily be customized, extended, and composed to create new conversational AI model architectures. \n",
+    "<br>  \n",
+    "For more information about NeMo, refer to the [NeMo product page](https://developer.nvidia.com/nvidia-nemo) and [documentation](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/starthere/intro.html). The open-source NeMo repository can be found [here](https://github.com/NVIDIA/NeMo)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ad02d1bb",
+   "metadata": {},
+   "source": [
+    "## Fine-tuning Riva NMT Multilingual model with NVIDIA NeMo\n",
+    "\n",
+    "For this tutorial, we will be fine-tuning the Riva NMT Multilingual Any-to-En model on the [Scielo](https://data.scielo.org/) [English-Spanish-Portugese dataset](https://figshare.com/articles/dataset/A_Large_Parallel_Corpus_of_Full-Text_Scientific_Articles/5382757).\n",
+    "\n",
+    "This tutorial covers fine-tuning only the NMT Multilingual model. Fine-tuning a Multilingual model is a relatively more challenging task (like choosing a balanced dataset covering multiple languages). At this stage, multilingual fine-tuning is only supported with specific NeMo and Pytorch lightning versions(PTL<2.0). We suggest you to use the specific NeMo branch as shared here.\n",
+    "\n",
+    "The process of fine-tuning here can be split into following steps:\n",
+    "1. Data download.\n",
+    "2. Data preprocessing.\n",
+    "3. Fine-tuning the NMT model with NeMo.\n",
+    "4. Evaluate the fine-tuned NMT model with NeMo.\n",
+    "5. Exporting the NeMo model\n",
+    "6. Deploying the fine-tuned NeMo NMT model on the Riva Speech Skills server.  \n",
+    "\n",
+    "Let's walk through each of these steps in detail."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "95eb024b",
+   "metadata": {},
+   "source": [
+    "<a id='nmt_requirements_and_setup'></a>\n",
+    "### Requirements and Setup\n",
+    "\n",
+    "This tutorial needs to be run from inside a NeMo docker container. If you are not running this tutorial through a NeMo docker container, please refer to the [Riva NMT Tutorials](https://ngc.nvidia.com/resources/riem1phmzvud:riva:riva_nmt_ea_tutorials)'s [README.md](https://ngc.nvidia.com/resources/riem1phmzvud:riva:riva_nmt_ea_tutorials/files?version=2.2.0-ea) to get started.\n",
```

Comment:
please check if these are valid links

Reply:
This link requires access. to Riva early access.

Reply:
this is old link from NMT EA days, we should point to latest link directly from github page now

Reply:
looks ok